src/style.css

```diff
@@ -1,3 +1,49 @@
 body {
-  margin: 0;
+  margin: 0 auto;
+  max-width: 1200px;
```

Comment:
max-width redundant here. 
body should be on full width of screen

Reply:
fixed